src/metabase/driver/common/parameters/values.clj

```diff
@@ -115,8 +118,9 @@
        (normalize-params matching-params))
      ;; otherwise, attempt to fall back to the default value specified as part of the template tag.
      (when-let [tag-default (:default tag)]
-       {:type  (:widget-type tag :dimension) ; widget-type is the actual type of the default value if set
-        :value tag-default})
+       (cond-> {:type    (:widget-type tag :dimension) ; widget-type is the actual type of the default value if set
+                :value   tag-default}
+         tag-opts (assoc :options tag-opts)))
```

Comment:
@metamben this code passes the options along with the template tag default value, to address your comment.

Reply:
Specifically, this code is for the case when there is no param value and a template tag supplies a default. It already existed, but I added the `:options` from the tag to the result, which is passed to `map->FieldFilter`.